style.css

```diff
@@ -106,12 +107,13 @@ img {
 
```

Comment:
Can we please add the line-heights as well? The paragraphs look too tight.

Reply:
added a variable of 35px line height and applied to paragraphs